Handle the no-sandbox option when spawning electron; display stdout, … (#37)
juliangruber/electron-stream: README.md

  - `basedir`: Set this if you need to require node modules in `node` mode
  - `static`: Serve static files from this directory at `/`
  - `loc`: a full url like `http://localhost:8080/` for using an existing http server. When `loc` is supplied, options `node`, `basedir`, and `static` are all ignored.

>   - `no-sandbox`: Run electron without the sandbox, required for debug information when using Docker

Reviewer: The API would be `sandbox`, `no-sandbox` is just for the CLI

>   - `sandbox`: Run electron with sandbox (default); use "--no-sandbox" to emit debug information when using Docker

Reviewer: ```suggestion
  - `sandbox`: Run electron with sandbox. Disable to emit debug information when using Docker. Defaults to `true`.
```

This section is describing the `API`, not the `CLI`, so it would be `sandbox: false`, not `--no-sandbox`. We don't have `CLI` documentation for this yet, so no need to document `--no-sandbox`.

>   - `no-sandbox`: Set this to run electron outside of a sandbox, may be required in a docker container to display stderr instead of a silent failure                                                              

Reviewer: ```suggestion
  - `no-sandbox`: Run electron without the sandbox, required for debug information when using Docker
```
